exec/cnex/stack.h

```diff
@@ -43,4 +43,22 @@ size_t callstack_top(struct tagTCallStack *stack);
 size_t callstack_peek(struct tagTCallStack *stack, int depth);
 int callstack_isEmpty(struct tagTCallStack *stack);
 
+
+typedef struct tagTItemStack {
```

Comment:
This seems to be declaring a bunch of things that are not defined and never used. Is there a purpose for this "item stack"?

Reply:
This was a preview for a stack designed for use with things like storing paths for %neonpath% and .neonpath parsing, etc.  I've decided on a linked list for that stuff anyway.  This was also designed to be used with the GC that I had been mocking up.  This code honestly shouldn't have been committed.

Reply:
Removed.